cb-talent-development/employer-style-base
# EM-1307: Header Large Alternate to Header Small (ESB)

**Purpose:**
> As a designer or a front end developer, I'd like the pages that use the "header_large_alternate" header partial to henceforth use the "header_small" partial (which might need to be created) so that going forward we have:
> - a defined set of headers
> - the same language for this header type
> - generic, reusable components when we want to use this header type again

This PR does the following:
- Removes styles associated with the partial found in Employer and merges them with the `header--small` mixin

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1307

**Changes:**
* Changes to setup
 * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * This may effect the following pages from EM-1308. Make sure they have a max width of 650px and 70px padding top and bottom for desktop; 50px padding top and bottom for mobile. Paragraph text should be 22px.
      * http://web.employer-4.development.c66.me/recruiting-solutions/display-advertising
      * http://web.employer-4.development.c66.me/recruiting-solutions/talent-driver
      * http://web.employer-4.development.c66.me/recruiting-solutions/source-and-screen
      * http://web.employer-4.development.c66.me/recruiting-solutions/hiring-events
      * http://web.employer-4.development.c66.me/employment-screening/drug-testing-services
      * http://web.employer-4.development.c66.me/employment-screening/electronic-i9-e-verify
      * http://web.employer-4.development.c66.me/recruiting-solutions/job-posting-and-candidate-profile-trends

**Screenshots**
* Before
![screen shot 2017-07-11 at 12 37 20 pm](https://user-images.githubusercontent.com/5348098/28081704-c502d6c8-6635-11e7-8eae-0e732f87d2bc.png)

* After
![screen shot 2017-07-11 at 12 36 46 pm](https://user-images.githubusercontent.com/5348098/28081711-ca4621da-6635-11e7-8c0b-3555bad768ab.png)

**QA Links:**
http://web.employer-4.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-4.development.c66.me/all-products-sitemap
  * NOTE: Our confluence doc also indicates that the page, http://web.employer-2.development.c66.me/employment-screening, also uses this partial. However, this page is redirected to Aurico which uses a partial that will be addressed in another Jira.

* Steps to take
  * Check the link above, the single header should have 70px top and bottom padding in desktop view. It should still have a max width of 650px. There should not be an empty paragraph container below the single header. On mobile, padding should be 50px top and bottom.

* Responsive considerations
  * N/A


**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/915

**Additional Information**
